src/plans/Stow.plp

```diff
@@ -0,0 +1,9 @@
+#include "plan-interface.h"
```

Comment:
There are a lot of small .plp files. Is this a Plexil requirement? Would the code be more navigable if they were merged into fewer files?

Reply:
I hear you!  Plexil is restricted to one plan (node) per file.  Also, the filename and node name should match.